common/LabelNumber/LabelNumber.css

```diff
@@ -0,0 +1,32 @@
+.wrapper {
+    display: inline-block;
+    text-align: center;
+    height: 38px;
+    width: 70px;
+    background: #d8e3e2;
```

Comment:
@nathan-bird Nope, this component does not need fixed width. Its initial use will be as a info element with 3 instances of the component (see #889 ), where I've suggested to use flex-grow property to make sure that if one of the instances should contain a very large number, it will squeeze the other instances to make room... Hope that makes sense :-)